test/liquid_voting/delegations_test.exs

```diff
@@ -82,6 +84,14 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:ok, %Delegation{} = delegation} = Delegations.create_delegation(args)
     end
 
+    # FAILING!!
+    # PR #109 https://github.com/liquidvotingio/api/issues/109
+    # "Allow creation of more than one proposal-specific delegation (different proposals) with same delegator and delegate"
+    # causes this failure.
+    # Since we wish to permit the existence of both a global and a proposal-specific delegation for the same delegate/delegator
+    # pair, in certain cases (see issue # 125: https://github.com/liquidvotingio/api/issues/125, for details), this test case
+    # should actually be acceptable (and this test, therefore, should be removed/ammended appropriately)
```

Comment:
Actually, this test covers duplicate global delegations? So should always fail. Unless I'm missing some implication from #109 here

Reply:
Yes, you are right. I need to change the comments to this test.

Reply:
I'm wondering if I should create a separate issue for this ("Prevent duplicate global delegations")?

Reply:
At this point I think it'll just add to the confusion :). I'd just wrap up this PR with more general comments, and get started on #125. I tried clarifying the constraints on slack and I'll post the conclusions on that issue later

Reply:
Can I just push the comment change for this test? (it's factually wrong at the moment)

Reply:
yeah, just axe any mention of future functionality. Just state that it's failing now pending new work on #125